apps/user/mutations.py

```diff
@@ -54,62 +59,62 @@ class ProfileCreateInputType(graphene.InputObjectType):
     signature = Upload(required=False)
 
 
-class ProfileUpdateInputType(graphene.InputObjectType):
+class UserUpdateInputType(graphene.InputObjectType):
     """
     Profile Update InputType
     """
     id = graphene.ID(required=True)
-    user = graphene.ID()
     phone_number = graphene.String()
     address = graphene.String()
     position = graphene.String()
 
-class CreateProfile(graphene.Mutation):
+
+class CreateUser(graphene.Mutation):
     class Arguments:
-        profile = ProfileCreateInputType(required=True)
-    
+        user = UserCreateInputType(required=True)
+
```

Comment:
Just asking, user creation will happen from `RegisterMutation` and `CreateUser` both?